sdk/communication/Azure.Communication.Identity/README.md

```diff
@@ -95,6 +95,14 @@ Console.WriteLine($"Token: {token}");
 Console.WriteLine($"Expires On: {expiresOn}");
 ```
 
+### Creating a user and a token in the same request
+```C# Snippet:CreateCommunicationUserAndToken
+Response<CommunicationUserIdentifierAndToken> response = client.CreateUserAndToken(scopes: new[] { CommunicationTokenScope.Chat });
+CommunicationUserIdentifierAndToken identifierAndToken = response.Value;
```

Comment:
you can also do `var (user, accessToken) = response.Value;`

Reply:
Cool thanks!